backend-project/small_eod/users/models.py

```diff
@@ -2,4 +2,8 @@
 
 
 class User(AbstractUser):
-    pass
+    def notify(self, sender, action, **kwargs):
+        pass
+
+    def get_enabled_notifications(self):
```

Comment:
Ja proponuje w tym miejscu zapisać implementacje tej metody to – na ten moment – jako `TemplateMailManager.TEMPLATE_MAP.keys()`.